src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+let data = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+data = [...data].map((item) => {
+  const population = item.innerText;
+  let populationNumber = '';
+
+  for (const ch of population) {
+    if (ch !== ',') {
+      populationNumber += ch;
+    }
+  }
```

Comment:
Using for loop inside of  `map` is bad for perfomence.
Instead try using regex for population